src/bin/client/stat.rs

```diff
@@ -171,6 +171,16 @@ fn collect_jobs(addr: &str, sub_m: &clap::ArgMatches<'_>) -> Vec<JobInfo> {
         })
         .collect();
 
+    // Decide if we need to remove jobs that aren't done
+    if sub_m.is_present("ONLY_DONE") {
+        jobs.retain(|j| {
+            match &j.status {
+                Status::Done {machine: _, output: _} => true,
```

Comment:
```suggestion
                Status::Done { machine: _, output: Some(_) } => true,
```

We don't really want jobs that have no output...